packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -289,20 +290,37 @@ export const TrackScreenDetailsTile = ({
     [track]
   )
 
-  const handlePressPlay = useCallback(() => {
-    if (isLineupLoading) return
-
-    if (isPlaying && isPlayingId) {
-      dispatch(tracksActions.pause())
-      recordPlay(track_id, false)
-    } else if (!isPlayingId) {
-      dispatch(tracksActions.play(uid))
-      recordPlay(track_id)
-    } else {
-      dispatch(tracksActions.play())
-      recordPlay(track_id)
-    }
-  }, [track_id, uid, isPlayingId, dispatch, isPlaying, isLineupLoading])
+  const play = useCallback(
+    ({ isPreview = false } = {}) => {
+      if (isLineupLoading) return
+
+      if (isPlaying && isPlayingId && isPreviewing === isPreview) {
+        dispatch(tracksActions.pause())
+        recordPlay(track_id, false)
+      } else if (!isPlayingId) {
+        dispatch(tracksActions.play(uid, { isPreview }))
+        recordPlay(track_id)
+      } else {
+        dispatch(tracksActions.play())
+        recordPlay(track_id)
+      }
+    },
+    [
+      track_id,
+      uid,
+      isPlayingId,
+      dispatch,
+      isPlaying,
+      isPreviewing,
+      isLineupLoading
+    ]
+  )
+
+  const handlePressPlay = useCallback(() => play({ isPreview: false }), [play])
```

Comment:
if you set up the default args above in line 294, do you need to explicitly pass `isPreview: false` here?

Reply:
Oh nope. Clearly I went through a couple iterations on this 😅